cmd/kyma/alpha/create/module/module.go

```diff
@@ -273,12 +276,39 @@ func (cmd *command) Run(ctx context.Context) error {
 		archiveFS = memoryfs.New()
 		l.Info("using in-memory archive")
 	}
-	// this builds the archive in memory, Alternatively one can store it on disk or in temp folder
-	archive, err := module.CreateArchive(archiveFS, cmd.opts.ModuleArchivePath, modDef)
-	if err != nil {
-		cmd.CurrentStep.Failure()
-		return err
+
+	var archive *comparch.ComponentArchive
+	gitPath, err := files.SearchForTargetDirByName(modDef.Source, ".git")
+	if gitPath == "" || err != nil {
+		l.Warnf("could not find git repository root, using %s directory", modDef.Source)
+		n := nice.NewNice()
+		n.PrintImportant("! CAUTION: The target folder is not a git repository. The sources will be not added to the layer")
```

Comment:
No need to use nice. Please log a Warning - but for user interaction please fail the step using `.Failuref("<some message to the user>")`

Reply:
I think using Nice here is good, please check `PrintImportant`, for interactive mode, it changes the message colour to better interact with user, it should be the default way if waiting for user input is needed.

But the problem here is this  `NewNice()` lack of initialised `NonInteractive` field, which is bad, I refactored in later commit.